SqlReplay.Console/BucketStats.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace SqlReplay.Console
+{
+    internal class BucketStats
+    {
+        public int timesFellBehind { get; set; }
+        public double minDelayms { get; set; }
+        public double maxDelayms { get; set; }
+        // count of stored procedured called
```

Comment:
Are these comments basically TODO items? or something else? Should they be removed?

Reply:
removed the comments and added a note to ADO Feature